src/Symfony/Component/Messenger/Transport/Serialization/Normalizer/FlattenExceptionNormalizer.php

```diff
@@ -78,6 +86,7 @@ public function denormalize(mixed $data, string $type, string $format = null, ar
 
     public function supportsDenormalization(mixed $data, string $type, string $format = null, array $context = []): bool
     {
+        // @deprecated remove $type check in 7.0
```

Comment:
let's remove the two deprecation note on this file, it's fine keeping as is imho